src/aspire/volume/volume.py

```diff
@@ -176,26 +181,43 @@ def __radd__(self, otherL):
 
     def __sub__(self, other):
         if isinstance(other, Volume):
-            res = Volume(self._data - other.asnumpy())
+            res = self.__class__(self._data - other.asnumpy())
         else:
-            res = Volume(self._data - other)
+            res = self.__class__(self._data - other)
 
         return res
 
     def __rsub__(self, otherL):
-        return Volume(otherL - self._data)
+        return self.__class__(otherL - self._data)
 
     def __mul__(self, other):
         if isinstance(other, Volume):
-            res = Volume(self._data * other.asnumpy())
+            res = self.__class__(self._data * other.asnumpy())
         else:
-            res = Volume(self._data * other)
+            res = self.__class__(self._data * other)
 
         return res
 
     def __rmul__(self, otherL):
         return self * otherL
 
+    def __truediv__(self, other):
+        """
+        Scalar division, follows numpy semantics.
+        """
+        if isinstance(other, Volume):
+            res = self.__class__(self._data / other.asnumpy())
+        else:
+            res = self.__class__(self._data / other)
+
+        return res
+
+    def __rtruedive__(self, otherL):
```

Comment:
A meaningful typo: `_rtruediv_`

Should we put this under testing?

Reply:
lol, another good deed.  Yeah, I'll fix this along with the other remark then add a test, thanks!